compiler/luci/service/src/HelperPads.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __LUCI_CIRCLE_SHAPE_INFERENCE_HELPER_PADS_H__
```

Comment:
guard name can be just file name or path/file name
this guard look strange but couldn't find it before.
let's use 
`__HELPER_PADS_H__` or 
`__LUCI_SERVICE_HELPER_PADS_H__`

Reply:
> guard name can be just file name or path/file name this guard looked strange but couldn't find it before. let's use `__HELPER_PADS_H__` (cause this file is only used in this folder) or `__LUCI_SERVICE_HELPER_PADS_H__`

`__LUCI_SERVICE_HELPER_PADS_H__` looks good to me.